Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Xuan <tanqazx@gmail.com>"]
 version = "0.1.9"
 
 [deps]
+BinaryDecisionDiagrams = "cd45f48d-2bff-4983-a793-756749635bd1"
```

Comment:
Since we're not using BDDs for now, maybe this can be removed?

Reply:
Yes, this is leftover of my experiments. I would love to look at them, but my idea was to start simply first and then to improve. To benchmark it properly. But then, I got a bit stuck how to deal with fluents. Do you know, if there is some implementation in other language, where I can get inspiration?

Reply:
Unfortunately I don't really know of existing implementations!

I think using BDDs to represent partial states is a bigger project that would require modifying PDDL.jl itself. Right now the representation for partial states is actually quite limited, in that it can't handle numeric fluents at all. So the `regress` and `relevant` functions don't work correctly for domains with fluents, which means that `BackwardPlanner` also does not work for domains with fluents. Addressing this would require some form of symbolic execution, such that numeric fluents can be assigned placeholder values. I would love to support something like that in the future, but it's not a priority unfortunately!